lib/active_fedora/relation/finder_methods.rb

```diff
@@ -163,6 +168,12 @@ def find_in_batches(conditions, opts = {})
       end
     end
 
+    # @deprecated
+    def find_in_batches(*args)
+      Deprecation.warn(ActiveFedora::Base, '.find_in_batches is deprecated; use .search_in_batches instead')
```

Comment:
Should we add that it'll be going away in 11.0.0 or some such indication of the *when*?